src/libecalc/core/models/pump/pump.py

```diff
@@ -212,6 +221,16 @@ def evaluate_rate_ps_pd_density(
         return pump_result
 
 
+def evaluate_streams(self, inlet_streams: List[Stream], outlet_stream: Stream) -> PumpModelResult:
+    total_requested_stream = Stream.mix_all(inlet_streams)
```

Comment:
Should we create a class `Streams` that we use to handle multiple streams, like a "collection/container class"? And change method name to `Streams.mix(streams)`? Not a big change though..

Reply:
We should consider it, we could also look at the difference between the Streams container and operational_settings passed to compressor and pump. We could only pass Streams, then add separate arguments to evaluate for pump and compressor if we need to pass something else than streams.